packages/skin/dist/number-input/number-input.css

```diff
@@ -0,0 +1,54 @@
+.number-input.textbox {
+    background: transparent;
+}
+
+.number-input label:first-child {
+    flex-grow: 1;
+    padding-left: var(--spacing-200);
+}
+
+.number-input label:first-child ~ input.textbox__control {
+    flex-grow: 0;
+}
+
+.number-input--show-delete .icon-btn.number-input__delete {
+    display: block;
+}
+
+.number-input--show-delete .icon-btn.number-input__decrement {
+    display: none;
+}
+
+.icon-btn.number-input__delete {
+    display: none;
+    margin-inline-end: calc(var(--spacing-100) * -1);
+    order: -1;
```

Comment:
do we need the `order` if display is `none`?

Reply:
We don't but it's better if the base styles have all the intended layout